pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,69 @@
+{ chafa
+, cmake
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, imagemagick_light
+, lib
+, libglvnd
+, libxcb
+, makeWrapper
+, ocl-icd
+, opencl-headers
+, pciutils
+, pkgconfig
+, stdenv
+, vulkan-loader
+, wayland
+, xfce
+, xorg
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastfetch";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "LinusDierheimer";
+    repo = pname;
+    rev = version;
+    hash = "sha256-OsK78irfO9r6pSEmlN6mFoKhQZgBqIYO8syj5AHH4IE=";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper pkgconfig ];
```

Comment:
```suggestion
  nativeBuildInputs = [ cmake makeWrapper pkg-config ];
```

Reply:
What's the difference between the two?

Reply:
AFAIK pkgconfig was an old name and is an alias. Maybe check aliases.nix?

Reply:
pkgconfig is an alias. ofborg should fail when it is used. Not sure why it isn't doing that.